src/main/java/org/folio/auth/authtokenmodule/MainVerticle.java

```diff
@@ -692,6 +692,7 @@ we generate a custom token (since we have that power) that
           endText(ctx, 401, "Invalid token: " + res.cause().getLocalizedMessage());
           return;
         }
+        ctx.response().putHeader(XOkapiHeaders.MODULE_TOKENS, moduleTokens.encode());
```

Comment:
Sure. I've added 3 identical comments.

pre and post filters unlike other module types are called even in case of errors (so they can log that error).